views/partials/password_tip_svg.hbs

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<svg role="img" width="87px" height="87px" viewBox="0 0 87 87" aria-labelledby="title" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+<svg role="img" width="87px" height="87px" viewBox="0 0 87 87" aria-label="Password tip number {{ math @index "+" 1 }}" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
I need to go through these things with a screen reader. From my limited a11y reading though, screen readers will read any text that is visible on the screen. My thinking here was that the aria-label  might help give the information some hierarchy/context. Also, {{ tip_title }} in the <title> tag may be redundant if screen readers pick it up in the <h4>anyway.

Reply:
Hhmm, some interesting info in https://css-tricks.com/accessible-svgs/#article-header-id-6

Reply:
Ok so maybe we just dispense with the title and label since the images are purely decorative and then raise an issue for improving the quality of other alt tags throughout.